### ISSUE TITLE
Remove Tab from Settings view

### ISSUE THREAD

**Issue by kaitmore** [DecipherNow/gm-fabric-dashboard#1471]
## Expected Behavior
Settings view should not have tabs

## Actual Behavior
Instance tab renders
![image](https://user-images.githubusercontent.com/22768918/35826011-7d08eaca-0a85-11e8-8370-a69899206278.png)

**Comment by spmcbride1201** [comment]
Yes, that's pretty nuts that we do that. I think we might need to experiment with shifting the setting page up or rendering an empty tab header to see what's the least visually jarring.
